resources/scss/findologic.scss

```diff
@@ -9,3 +9,4 @@
 @import 'views/ItemList/Components/Filter/item-range-slider';
 @import 'views/ItemList/Components/item-search';
 @import 'views/Category/Item/Partials/promotion';
+@import 'nouislider/dist/nouislider';
```

Comment:
If I remember correctly I had to manually import the css for this library because it didnt function like it did the develop branch, Ill check it out a bit more why this is an issue